### ISSUE TITLE
Replace the `-xf-` CSS prefix with `-maui-`

### ISSUE THREAD

**Issue by mattleibow** [dotnet/upgrade-assistant#759]
## Summary

Xamarin.Forms used a `-xf-` prefix for the CSS support. Maui should probably use `-maui-` instead.

## Motivation and goals

This is just a name change because Maui is no longer Xamarin.Forms, so it is good to unify the new names.

## In scope

Replace `-xf-` prefies with `-maui-` in CSS files.

## Out of scope

Scenarios you explicitly want to exclude.

## Risks / unknowns

There might be a case where some third party library has already taken the prefix `-maui-` for a Xamarin.Forms app, but this is highly unlikely.

## Examples

**.css**
```css
button {
  -xf-placeholder-color: red;
}
```

**.xaml**
```xaml
<ContentPage>
    <ContentPage.Resources>
        <StyleSheet>
            <![CDATA[
button {
  -xf-placeholder-color: red;
}
            ]]>
        </StyleSheet>
    </ContentPage.Resources>
</ContentPage>
```

**Comment by sunandabalu** [comment]
@Sweekriti91 Could you review this design-proposal?

**Comment by Sweekriti91** [comment]
@mattleibow would the changes be isolated only to CSS files? or are changes expected in any C# or XAML files too?

**Comment by mattleibow** [comment]
I see we support css in Xaml under the resources, do that might need to be migrated too

**Comment by Sweekriti91** [comment]
@mattleibow  sounds good! I think this makes sense to add after the MVP of the MAUI extension work is complete. There are implementations I'm working on that can help with this issue. You can open a PR to add this support once my extension work is complete.
